packaging/nuget/NServiceBus.RabbitMQ.nuspec

```diff
@@ -16,7 +16,7 @@
     <tags>nservicebus servicebus msmq cqrs publish subscribe</tags>
     <dependencies>
       <dependency id="NServiceBus" version="[6.0.0, 7.0.0)" />
-      <dependency id="RabbitMQ.Client" version="[4.1.0, 5.1.0)" />
+      <dependency id="RabbitMQ.Client" version="[5.0.1, 5.1.0)" />
```

Comment:
I went ahead and bumped the client up to 5.0.1. I figured it made sense to go ahead and do it as part of the hotfix since we know there's the bug in 4.1.0. What do you think? I can always remove the commit if we don't want this.

Reply:
@bording I think it makes sense to have this in the patch, but should we have a "bug" to link to in the release notes, since it's unrelated to both the current bugs that we have raised?